fe/fe-core/src/main/java/org/apache/doris/fsv2/FileSystemFactory.java

```diff
@@ -52,6 +53,22 @@ public static RemoteFileSystem get(String name, Map<String, String> properties)
         return new BrokerFileSystem(name, properties);
     }
 
+    public static RemoteFileSystem get(FileSystemType fileSystemType, Map<String, String> properties,
```

Comment:
I think in this class, we should only keep `get(StorageProperties storageProperties)`, and remove all other `get` method.